src/main/kotlin/org/quiltmc/community/modes/quilt/extensions/UtilityExtension.kt

```diff
@@ -154,6 +160,57 @@ class UtilityExtension : Extension() {
             }
         }
 
+        event<InteractionCreateEvent> {
```

Comment:
You should really use [the components system](https://github.com/Kord-Extensions/kord-extensions/blob/develop/kord-extensions/src/test/kotlin/com/kotlindiscord/kord/extensions/test/bot/TestExtension.kt#L277) for this, especially since guessable button IDs are insecure,

Note: The example above doesn't supply a timeout - you definitely want to.

You can use a `lateinit var` to store the sent message if you want to edit it later.